components/pages/MyGarden.js

```diff
@@ -1,25 +1,50 @@
-import React from 'react';
-import { View, Text, ImageBackground, SafeAreaView} from 'react-native';
+import React, { Component } from 'react';
+import { View, Text, ImageBackground, SafeAreaView, ScrollView} from 'react-native';
```

Comment:
Getting that scroll view in there so we can see all the little plant babies!

Reply:
It was surprisingly easy to implement.